dotnet/samples/README.md

```diff
@@ -1,9 +1,9 @@
-## Kernel Samples
+## Semantic Kernel Samples
 
-| Type                                            | Description                                                                                                     |
-| ----------------------------------------------- | --------------------------------------------------------------------------------------------------------------- |
-| [`Getting Started`](./GettingStarted/README.md) | Take this step by step tutorial to get started with the Semantic Kernel and get introduced to the key concepts. |
-| [`Concepts`](./Concepts/README.md)              | This section contains focussed samples which illustrate all of the concepts included in the Semantic Kernel.    |
-| [`Demos`](./Demos/README.md)                    | Look here to find a sample which demonstrate how to use many of Semantic Kernel features.                       |
-| [`LearnResources`](./LearnResources/README.md)  | Code snippets that are related to online documentation sources like Microsoft Learn, DevBlogs and others        |
-| <strike>`KernelSyntaxExamples`</strike>         | ⚠️ Work in progress: Moving into `Concepts`.                                                                    |
+| Type                                                               | Description                                                                                                            |
+| ------------------------------------------------------------------ | ---------------------------------------------------------------------------------------------------------------------- |
+| [`GettingStarted`](./GettingStarted/README.md)                     | Take this step by step tutorial to get started with the Semantic Kernel and get introduced to the key concepts.        |
+| [`GettingStartedWithAgents`](./GettingStartedWithAgents/README.md) | Take this step by step tutorial to get started with the Semantic Kernel Agents and get introduced to the key concepts. |
+| [`Concepts`](./Concepts/README.md)                                 | This section contains focussed samples which illustrate all of the concepts included in the Semantic Kernel.           |
+| [`Demos`](./Demos/README.md)                                       | Look here to find a sample which demonstrate how to use many of Semantic Kernel features.                              |
+| [`LearnResources`](./LearnResources/README.md)                     | Code snippets that are related to online documentation sources like Microsoft Learn, DevBlogs and others               |
```

Comment:
nit: Adding the extra whitespace means that as soon as someone adds a wider column they need to update every row and then reviewing that change requires the reviewer to turn off whitespace. The extra whitespace isn't needed for proper display of the table.

Reply:
Ok, will avoid that in the future, my formatter does that automatically 👍🏻